portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,30 +22,52 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> messages;
-
-  @Override
-  public void init() {
-    messages = new ArrayList<>();
-  }
+  //constants for Datastore entity and property names
+  private final String COMMENT = "Comment";
+  private final String CONTENT = "content";
```

Comment:
nit: use more descriptive names for the constants, e.g. `ENTITY_COMMENT` and `PROPERTY_CONTENT`

(it's verbose but makes it more clear what the constants represent)

Reply:
oh and also make these "static" too